http-server-tck/src/main/java/io/micronaut/http/server/tck/tests/constraintshandler/ControllerConstraintHandlerTest.java

```diff
@@ -0,0 +1,273 @@
+
+import jakarta.validation.ConstraintViolationException;
+import jakarta.validation.Valid;
+import jakarta.validation.constraints.Email;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
+import java.io.IOException;
+import java.util.Map;
+import java.util.Optional;
+import io.micronaut.http.tck.AssertionUtils;
+import io.micronaut.http.tck.HttpResponseAssertion;
+import io.micronaut.http.tck.ServerUnderTest;
+import io.micronaut.http.tck.ServerUnderTestProviderUtils;
+import static io.micronaut.http.tck.TestScenario.asserts;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@SuppressWarnings({
+    "java:S5960", // We're allowed assertions, as these are used in tests only
+    "checkstyle:MissingJavadocType",
+    "checkstyle:DesignForExtension"
+})
+public class ControllerConstraintHandlerTest {
+    public static final String SPEC_NAME = "ControllerConstraintHandlerTest";
+    private static final HttpResponseAssertion TEAPOT_ASSERTION = HttpResponseAssertion.builder()
+        .status(HttpStatus.I_AM_A_TEAPOT)
+        .assertResponse(response -> {
+            Optional<String> json = response.getBody(Argument.of(String.class));
+            assertTrue(json.isPresent());
+            assertTrue(json.get().contains("secret"));
+            assertTrue(json.get().contains("password"));
+        })
+        .build();
+
+    @Test
+    void testPojoWithNullable() throws IOException {
+        asserts(SPEC_NAME,
+            HttpRequest.POST("/constraints-via-handler", "{\"username\":\"tim@micronaut.example\",\"password\":\"secret\"}"),
+            (server, request) -> AssertionUtils.assertDoesNotThrow(server, request, HttpResponseAssertion.builder()
+                .status(HttpStatus.OK)
+                .build()));
+        asserts(SPEC_NAME,
+            HttpRequest.POST("/constraints-via-handler/with-at-nullable", "{\"username\":\"invalidemail\",\"password\":\"secret\"}"),
+            (server, request) -> AssertionUtils.assertThrows(server, request, constraintAssertion("must be a well-formed email address")));
+        asserts(SPEC_NAME,
+            HttpRequest.POST("/constraints-via-handler/with-at-nullable", "{\"username\":\"\",\"password\":\"secret\"}"),
+            (server, request) -> AssertionUtils.assertThrows(server, request, constraintAssertion("must not be blank\"")));
+        asserts(SPEC_NAME,
+            HttpRequest.POST("/constraints-via-on-error-method/with-at-nullable", "{\"username\":\"\",\"password\":\"secret\"}"),
+            (server, request) -> AssertionUtils.assertThrows(server, request, TEAPOT_ASSERTION));
+
+        asserts(SPEC_NAME,
+            HttpRequest.POST("/constraints-via-on-error-method/with-at-nullable", "{\"password\":\"secret\"}"),
+            (server, request) -> AssertionUtils.assertThrows(server, request, TEAPOT_ASSERTION));
+
+    }
+
+    @Test
+    void testWithPojoWithoutAnnotations() throws IOException {
```

Comment:
this tests passes in Micronaut Framework 3, however it fails in Micronaut Framework 4 with: 

```
12:28:14.290 [Test worker] INFO  i.m.context.env.DefaultEnvironment - Established active environments: [test]
12:28:14.309 [default-nioEventLoopGroup-6-2] DEBUG i.m.h.client.netty.DefaultHttpClient - Sending HTTP POST to http://localhost:62122/constraints-via-handler
12:28:14.309 [default-nioEventLoopGroup-6-2] TRACE i.m.h.client.netty.DefaultHttpClient - host: localhost:62122
12:28:14.309 [default-nioEventLoopGroup-6-2] TRACE i.m.h.client.netty.DefaultHttpClient - connection: keep-alive
12:28:14.309 [default-nioEventLoopGroup-6-2] TRACE i.m.h.client.netty.DefaultHttpClient - content-type: application/json
12:28:14.309 [default-nioEventLoopGroup-6-2] TRACE i.m.h.client.netty.DefaultHttpClient - content-length: 47
12:28:14.309 [default-nioEventLoopGroup-6-2] TRACE i.m.h.client.netty.DefaultHttpClient - Request Body
12:28:14.309 [default-nioEventLoopGroup-6-2] TRACE i.m.h.client.netty.DefaultHttpClient - ----
12:28:14.309 [default-nioEventLoopGroup-6-2] TRACE i.m.h.client.netty.DefaultHttpClient - {"username":"invalidemail","password":"secret"}
12:28:14.309 [default-nioEventLoopGroup-6-2] TRACE i.m.h.client.netty.DefaultHttpClient - ----
12:28:14.315 [default-nioEventLoopGroup-6-2] DEBUG i.m.h.client.netty.DefaultHttpClient - Received response 400 from http://localhost:62122/constraints-via-handler
12:28:14.315 [default-nioEventLoopGroup-6-2] TRACE i.m.h.client.netty.DefaultHttpClient - HTTP Client Response Received (400 Bad Request) for Request: POST http://localhost:62122/constraints-via-handler
12:28:14.315 [default-nioEventLoopGroup-6-2] TRACE i.m.h.client.netty.DefaultHttpClient - date: Wed, 17 May 2023 10:28:14 GMT
12:28:14.315 [default-nioEventLoopGroup-6-2] TRACE i.m.h.client.netty.DefaultHttpClient - Content-Type: application/json
12:28:14.315 [default-nioEventLoopGroup-6-2] TRACE i.m.h.client.netty.DefaultHttpClient - content-length: 194
12:28:14.316 [default-nioEventLoopGroup-6-2] TRACE i.m.h.client.netty.DefaultHttpClient - Response Body
12:28:14.316 [default-nioEventLoopGroup-6-2] TRACE i.m.h.client.netty.DefaultHttpClient - ----
12:28:14.316 [default-nioEventLoopGroup-6-2] TRACE i.m.h.client.netty.DefaultHttpClient - {"message":"Bad Request","_embedded":{"errors":[{"message":"credentials.username: must be a well-formed email address"}]},"_links":{"self":{"href":"/constraints-via-handler","templated":false}}}
12:28:14.316 [default-nioEventLoopGroup-6-2] TRACE i.m.h.client.netty.DefaultHttpClient - ----
12:28:14.327 [Test worker] INFO  i.m.context.env.DefaultEnvironment - Established active environments: [test]
12:28:14.342 [default-nioEventLoopGroup-7-2] DEBUG i.m.h.client.netty.DefaultHttpClient - Sending HTTP POST to http://localhost:62124/constraints-via-handler
12:28:14.343 [default-nioEventLoopGroup-7-2] TRACE i.m.h.client.netty.DefaultHttpClient - host: localhost:62124
12:28:14.343 [default-nioEventLoopGroup-7-2] TRACE i.m.h.client.netty.DefaultHttpClient - connection: keep-alive
12:28:14.343 [default-nioEventLoopGroup-7-2] TRACE i.m.h.client.netty.DefaultHttpClient - content-type: application/json
12:28:14.343 [default-nioEventLoopGroup-7-2] TRACE i.m.h.client.netty.DefaultHttpClient - content-length: 35
12:28:14.343 [default-nioEventLoopGroup-7-2] TRACE i.m.h.client.netty.DefaultHttpClient - Request Body
12:28:14.343 [default-nioEventLoopGroup-7-2] TRACE i.m.h.client.netty.DefaultHttpClient - ----
12:28:14.343 [default-nioEventLoopGroup-7-2] TRACE i.m.h.client.netty.DefaultHttpClient - {"username":"","password":"secret"}
12:28:14.343 [default-nioEventLoopGroup-7-2] TRACE i.m.h.client.netty.DefaultHttpClient - ----
12:28:14.350 [default-nioEventLoopGroup-7-2] DEBUG i.m.h.client.netty.DefaultHttpClient - Received response 400 from http://localhost:62124/constraints-via-handler
12:28:14.350 [default-nioEventLoopGroup-7-2] TRACE i.m.h.client.netty.DefaultHttpClient - HTTP Client Response Received (400 Bad Request) for Request: POST http://localhost:62124/constraints-via-handler
12:28:14.350 [default-nioEventLoopGroup-7-2] TRACE i.m.h.client.netty.DefaultHttpClient - date: Wed, 17 May 2023 10:28:14 GMT
12:28:14.350 [default-nioEventLoopGroup-7-2] TRACE i.m.h.client.netty.DefaultHttpClient - Content-Type: application/json
12:28:14.350 [default-nioEventLoopGroup-7-2] TRACE i.m.h.client.netty.DefaultHttpClient - content-length: 248
12:28:14.350 [default-nioEventLoopGroup-7-2] TRACE i.m.h.client.netty.DefaultHttpClient - Response Body
12:28:14.350 [default-nioEventLoopGroup-7-2] TRACE i.m.h.client.netty.DefaultHttpClient - ----
12:28:14.350 [default-nioEventLoopGroup-7-2] TRACE i.m.h.client.netty.DefaultHttpClient - {"message":"Bad Request","_embedded":{"errors":[{"message":"credentials.username: must be a well-formed email address"},{"message":"credentials.username: must not be blank"}]},"_links":{"self":{"href":"/constraints-via-handler","templated":false}}}
12:28:14.350 [default-nioEventLoopGroup-7-2] TRACE i.m.h.client.netty.DefaultHttpClient - ----
12:28:14.362 [Test worker] INFO  i.m.context.env.DefaultEnvironment - Established active environments: [test]
12:28:14.377 [default-nioEventLoopGroup-8-2] DEBUG i.m.h.client.netty.DefaultHttpClient - Sending HTTP POST to http://localhost:62126/constraints-via-on-error-method
12:28:14.377 [default-nioEventLoopGroup-8-2] TRACE i.m.h.client.netty.DefaultHttpClient - host: localhost:62126
12:28:14.377 [default-nioEventLoopGroup-8-2] TRACE i.m.h.client.netty.DefaultHttpClient - connection: keep-alive
12:28:14.377 [default-nioEventLoopGroup-8-2] TRACE i.m.h.client.netty.DefaultHttpClient - content-type: application/json
12:28:14.377 [default-nioEventLoopGroup-8-2] TRACE i.m.h.client.netty.DefaultHttpClient - content-length: 35
12:28:14.377 [default-nioEventLoopGroup-8-2] TRACE i.m.h.client.netty.DefaultHttpClient - Request Body
12:28:14.377 [default-nioEventLoopGroup-8-2] TRACE i.m.h.client.netty.DefaultHttpClient - ----
12:28:14.377 [default-nioEventLoopGroup-8-2] TRACE i.m.h.client.netty.DefaultHttpClient - {"username":"","password":"secret"}
12:28:14.377 [default-nioEventLoopGroup-8-2] TRACE i.m.h.client.netty.DefaultHttpClient - ----
12:28:14.381 [default-nioEventLoopGroup-8-3] ERROR i.m.http.server.RouteExecutor - Unexpected error occurred: Already claimed
java.lang.IllegalStateException: Already claimed
	at io.micronaut.http.server.netty.body.ManagedBody.value(ManagedBody.java:47)
	at io.micronaut.http.server.netty.body.ImmediateSingleObjectBody.valueUnclaimed(ImmediateSingleObjectBody.java:81)
	at io.micronaut.http.server.netty.NettyHttpRequest.getBody(NettyHttpRequest.java:399)
	at io.micronaut.http.server.netty.NettyHttpRequest.getBody(NettyHttpRequest.java:416)
	at io.micronaut.http.HttpMessage.getBody(HttpMessage.java:94)
	at io.micronaut.http.server.netty.NettyHttpRequest.getBody(NettyHttpRequest.java:410)
	at io.micronaut.http.server.tck.tests.constraintshandler.ControllerConstraintHandlerTest$OnErrorMethodController.constraintsEx(ControllerConstraintHandlerTest.java:193)
	at io.micronaut.http.server.tck.tests.constraintshandler.$ControllerConstraintHandlerTest$OnErrorMethodController$Definition$Exec.dispatch(Unknown Source)
	at io.micronaut.context.AbstractExecutableMethodsDefinition$DispatchedExecutableMethod.invoke(AbstractExecutableMethodsDefinition.java:442)
	at io.micronaut.context.DefaultBeanContext$BeanExecutionHandle.invoke(DefaultBeanContext.java:3858)
	at io.micronaut.web.router.AbstractRouteMatch.execute(AbstractRouteMatch.java:230)
	at io.micronaut.http.context.ServerRequestContext.with(ServerRequestContext.java:103)
	at io.micronaut.http.server.RouteExecutor.executeRouteAndConvertBody(RouteExecutor.java:437)
	at io.micronaut.http.server.RouteExecutor.callRoute(RouteExecutor.java:428)
	at io.micronaut.http.server.RequestLifecycle.lambda$onErrorNoFilter$6(RequestLifecycle.java:184)
	at io.micronaut.core.execution.ImperativeExecutionFlowImpl.flatMap(ImperativeExecutionFlowImpl.java:72)
	at io.micronaut.http.server.RequestLifecycle.onErrorNoFilter(RequestLifecycle.java:184)
	at io.micronaut.core.execution.ImperativeExecutionFlowImpl.onErrorResume(ImperativeExecutionFlowImpl.java:112)
	at io.micronaut.http.server.RequestLifecycle.lambda$normalFlow$3(RequestLifecycle.java:149)
	at io.micronaut.http.server.RequestLifecycle.lambda$runWithFilters$13(RequestLifecycle.java:262)
	at io.micronaut.http.filter.FilterRunner.processRequestFilter(FilterRunner.java:279)
	at io.micronaut.http.filter.FilterRunner.filterRequest0(FilterRunner.java:169)
	at io.micronaut.http.filter.FilterRunner.lambda$filterRequest0$3(FilterRunner.java:169)
	at io.micronaut.http.filter.FilterRunner$FilterChainImpl.proceed(FilterRunner.java:859)
	at io.micronaut.management.endpoint.EndpointsFilter.doFilterOnce(EndpointsFilter.java:71)
	at io.micronaut.http.filter.OncePerRequestHttpServerFilter.doFilter(OncePerRequestHttpServerFilter.java:41)
	at io.micronaut.http.filter.HttpServerFilter.doFilter(HttpServerFilter.java:48)
	at io.micronaut.http.filter.FilterRunner.processRequestFilter(FilterRunner.java:257)
	at io.micronaut.http.filter.FilterRunner.filterRequest0(FilterRunner.java:169)
	at io.micronaut.http.filter.FilterRunner.lambda$filterRequest0$3(FilterRunner.java:169)
	at io.micronaut.http.filter.FilterRunner.processRequestFilter(FilterRunner.java:228)
	at io.micronaut.http.filter.FilterRunner.filterRequest0(FilterRunner.java:169)
	at io.micronaut.http.filter.FilterRunner.lambda$filterRequest0$3(FilterRunner.java:169)
	at io.micronaut.core.execution.ImperativeExecutionFlowImpl.flatMap(ImperativeExecutionFlowImpl.java:72)
	at io.micronaut.http.filter.FilterRunner.processRequestFilter(FilterRunner.java:250)
	at io.micronaut.http.filter.FilterRunner.filterRequest0(FilterRunner.java:169)
	at io.micronaut.http.filter.FilterRunner.filterRequest(FilterRunner.java:153)
	at io.micronaut.http.filter.FilterRunner.run(FilterRunner.java:148)
	at io.micronaut.http.server.RequestLifecycle.runWithFilters(RequestLifecycle.java:279)
	at io.micronaut.http.server.RequestLifecycle.normalFlow(RequestLifecycle.java:146)
	at io.micronaut.http.server.netty.NettyRequestLifecycle.handleNormal(NettyRequestLifecycle.java:87)
	at io.micronaut.http.server.netty.RoutingInBoundHandler.accept(RoutingInBoundHandler.java:212)
	at io.micronaut.http.server.netty.websocket.NettyServerWebSocketUpgradeHandler.accept(NettyServerWebSocketUpgradeHandler.java:150)
	at io.micronaut.http.server.netty.handler.PipeliningServerHandler$OptimisticBufferingInboundHandler.read(PipeliningServerHandler.java:410)
	at io.micronaut.http.server.netty.handler.PipeliningServerHandler.channelRead(PipeliningServerHandler.java:203)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.channel.ChannelInboundHandlerAdapter.channelRead(ChannelInboundHandlerAdapter.java:93)
	at io.netty.handler.codec.http.websocketx.extensions.WebSocketServerExtensionHandler.channelRead(WebSocketServerExtensionHandler.java:109)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:444)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.handler.codec.MessageToMessageDecoder.channelRead(MessageToMessageDecoder.java:103)
	at io.netty.handler.codec.MessageToMessageCodec.channelRead(MessageToMessageCodec.java:111)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.channel.ChannelInboundHandlerAdapter.channelRead(ChannelInboundHandlerAdapter.java:93)
	at io.netty.handler.codec.http.HttpServerKeepAliveHandler.channelRead(HttpServerKeepAliveHandler.java:64)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.channel.CombinedChannelDuplexHandler$DelegatingChannelHandlerContext.fireChannelRead(CombinedChannelDuplexHandler.java:436)
	at io.netty.handler.codec.ByteToMessageDecoder.fireChannelRead(ByteToMessageDecoder.java:346)
	at io.netty.handler.codec.ByteToMessageDecoder.channelRead(ByteToMessageDecoder.java:318)
	at io.netty.channel.CombinedChannelDuplexHandler.channelRead(CombinedChannelDuplexHandler.java:251)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.handler.timeout.IdleStateHandler.channelRead(IdleStateHandler.java:286)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:442)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:412)
	at io.netty.channel.DefaultChannelPipeline$HeadContext.channelRead(DefaultChannelPipeline.java:1410)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:440)
	at io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:420)
	at io.netty.channel.DefaultChannelPipeline.fireChannelRead(DefaultChannelPipeline.java:919)
	at io.netty.channel.nio.AbstractNioByteChannel$NioByteUnsafe.read(AbstractNioByteChannel.java:166)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:788)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:724)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:650)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:562)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:997)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:833)
12:28:14.386 [default-nioEventLoopGroup-8-2] DEBUG i.m.h.client.netty.DefaultHttpClient - Received response 500 from http://localhost:62126/constraints-via-on-error-method
12:28:14.386 [default-nioEventLoopGroup-8-2] TRACE i.m.h.client.netty.DefaultHttpClient - HTTP Client Response Received (500 Internal Server Error) for Request: POST http://localhost:62126/constraints-via-on-error-method
12:28:14.386 [default-nioEventLoopGroup-8-2] TRACE i.m.h.client.netty.DefaultHttpClient - date: Wed, 17 May 2023 10:28:14 GMT
12:28:14.387 [default-nioEventLoopGroup-8-2] TRACE i.m.h.client.netty.DefaultHttpClient - Content-Type: application/json
12:28:14.387 [default-nioEventLoopGroup-8-2] TRACE i.m.h.client.netty.DefaultHttpClient - content-length: 193
12:28:14.387 [default-nioEventLoopGroup-8-2] TRACE i.m.h.client.netty.DefaultHttpClient - Response Body
12:28:14.387 [default-nioEventLoopGroup-8-2] TRACE i.m.h.client.netty.DefaultHttpClient - ----
12:28:14.387 [default-nioEventLoopGroup-8-2] TRACE i.m.h.client.netty.DefaultHttpClient - {"message":"Internal Server Error","_embedded":{"errors":[{"message":"Internal Server Error: Already claimed"}]},"_links":{"self":{"href":"/constraints-via-on-error-method","templated":false}}}
12:28:14.387 [default-nioEventLoopGroup-8-2] TRACE i.m.h.client.netty.DefaultHttpClient - ----

expected: <I_AM_A_TEAPOT> but was: <INTERNAL_SERVER_ERROR>
Expected :I_AM_A_TEAPOT
Actual   :INTERNAL_SERVER_ERROR
<Click to see difference>

org.opentest4j.AssertionFailedError: expected: <I_AM_A_TEAPOT> but was: <INTERNAL_SERVER_ERROR>
	at app//org.junit.jupiter.api.AssertionFailureBuilder.build(AssertionFailureBuilder.java:151)
	at app//org.junit.jupiter.api.AssertionFailureBuilder.buildAndThrow(AssertionFailureBuilder.java:132)
	at app//org.junit.jupiter.api.AssertEquals.failNotEqual(AssertEquals.java:197)
	at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:182)
	at app//org.junit.jupiter.api.AssertEquals.assertEquals(AssertEquals.java:177)
	at app//org.junit.jupiter.api.Assertions.assertEquals(Assertions.java:1142)
	at app//io.micronaut.http.tck.AssertionUtils.assertThrows(AssertionUtils.java:67)
	at app//io.micronaut.http.server.tck.tests.constraintshandler.ControllerConstraintHandlerTest.lambda$testWithPojoWithoutAnnotations$8(ControllerConstraintHandlerTest.java:104)
	at app//io.micronaut.http.tck.TestScenario.run(TestScenario.java:118)
	at app//io.micronaut.http.tck.TestScenario$Builder.run(TestScenario.java:201)
	at app//io.micronaut.http.tck.TestScenario.asserts(TestScenario.java:104)
	at app//io.micronaut.http.server.tck.tests.constraintshandler.ControllerConstraintHandlerTest.testWithPojoWithoutAnnotations(ControllerConstraintHandlerTest.java:102)
	at java.base@17.0.5/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base@17.0.5/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base@17.0.5/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base@17.0.5/java.lang.reflect.Method.invoke(Method.java:568)
	at app//org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:727)
	at app//org.junit.jupiter.engine.execution.MethodInvocation.proceed(MethodInvocation.java:60)
	at app//org.junit.jupiter.engine.execution.InvocationInterceptorChain$ValidatingInvocation.proceed(InvocationInterceptorChain.java:131)
	at app//org.junit.jupiter.engine.extension.TimeoutExtension.intercept(TimeoutExtension.java:156)
	at app//org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestableMethod(TimeoutExtension.java:147)
	at app//org.junit.jupiter.engine.extension.TimeoutExtension.interceptTestMethod(TimeoutExtension.java:86)
	at app//org.junit.jupiter.engine.execution.InterceptingExecutableInvoker$ReflectiveInterceptorCall.lambda$ofVoidMethod$0(InterceptingExecutableInvoker.java:103)
	at app//org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.lambda$invoke$0(InterceptingExecutableInvoker.java:93)
	at app//org.junit.jupiter.engine.execution.InvocationInterceptorChain$InterceptedInvocation.proceed(InvocationInterceptorChain.java:106)
	at app//org.junit.jupiter.engine.execution.InvocationInterceptorChain.proceed(InvocationInterceptorChain.java:64)
	at app//org.junit.jupiter.engine.execution.InvocationInterceptorChain.chainAndInvoke(InvocationInterceptorChain.java:45)
	at app//org.junit.jupiter.engine.execution.InvocationInterceptorChain.invoke(InvocationInterceptorChain.java:37)
	at app//org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:92)
	at app//org.junit.jupiter.engine.execution.InterceptingExecutableInvoker.invoke(InterceptingExecutableInvoker.java:86)
	at app//org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$7(TestMethodTestDescriptor.java:217)
	at app//org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at app//org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:213)
	at app//org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:138)
	at app//org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:68)
	at app//org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:151)
	at app//org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at app//org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at app//org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at app//org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at app//org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at app//org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at app//org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base@17.0.5/java.util.ArrayList.forEach(ArrayList.java:1511)
	at app//org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at app//org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at app//org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at app//org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at app//org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at app//org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at app//org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at app//org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at app//org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at java.base@17.0.5/java.util.ArrayList.forEach(ArrayList.java:1511)
	at app//org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:41)
	at app//org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$6(NodeTestTask.java:155)
	at app//org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at app//org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$8(NodeTestTask.java:141)
	at app//org.junit.platform.engine.support.hierarchical.Node.around(Node.java:137)
	at app//org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$9(NodeTestTask.java:139)
	at app//org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at app//org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:138)
	at app//org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:95)
	at app//org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at app//org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at app//org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at app//org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:147)
	at app//org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:127)
	at app//org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:90)
	at app//org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:73)
	at app//org.junit.platform.suite.engine.SuiteLauncher.execute(SuiteLauncher.java:63)
	at app//org.junit.platform.suite.engine.SuiteTestDescriptor.execute(SuiteTestDescriptor.java:128)
	at app//org.junit.platform.suite.engine.SuiteTestEngine.lambda$execute$0(SuiteTestEngine.java:73)
	at java.base@17.0.5/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:183)
	at java.base@17.0.5/java.util.stream.ReferencePipeline$3$1.accept(ReferencePipeline.java:197)
	at java.base@17.0.5/java.util.Iterator.forEachRemaining(Iterator.java:133)
	at java.base@17.0.5/java.util.Spliterators$IteratorSpliterator.forEachRemaining(Spliterators.java:1845)
	at java.base@17.0.5/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:509)
	at java.base@17.0.5/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base@17.0.5/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:150)
	at java.base@17.0.5/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:173)
	at java.base@17.0.5/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base@17.0.5/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
	at app//org.junit.platform.suite.engine.SuiteTestEngine.execute(SuiteTestEngine.java:73)
	at app//org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:147)
	at app//org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:127)
	at app//org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:90)
	at app//org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:55)
	at app//org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:102)
	at app//org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:54)
	at app//org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:114)
	at app//org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:86)
	at app//org.junit.platform.launcher.core.DefaultLauncherSession$DelegatingLauncher.execute(DefaultLauncherSession.java:86)
	at org.gradle.api.internal.tasks.testing.junitplatform.JUnitPlatformTestClassProcessor$CollectAllTestClassesExecutor.processAllTestClasses(JUnitPlatformTestClassProcessor.java:110)
	at org.gradle.api.internal.tasks.testing.junitplatform.JUnitPlatformTestClassProcessor$CollectAllTestClassesExecutor.access$000(JUnitPlatformTestClassProcessor.java:90)
	at org.gradle.api.internal.tasks.testing.junitplatform.JUnitPlatformTestClassProcessor.stop(JUnitPlatformTestClassProcessor.java:85)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.stop(SuiteTestClassProcessor.java:62)
	at java.base@17.0.5/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base@17.0.5/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base@17.0.5/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base@17.0.5/java.lang.reflect.Method.invoke(Method.java:568)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
	at jdk.proxy1/jdk.proxy1.$Proxy2.stop(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker$3.run(TestWorker.java:193)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.executeAndMaintainThreadName(TestWorker.java:129)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:100)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:60)
	at org.gradle.process.internal.worker.child.ActionExecutionWorker.execute(ActionExecutionWorker.java:56)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:113)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:65)
	at app//worker.org.gradle.process.internal.worker.GradleWorkerMain.run(GradleWorkerMain.java:69)
	at app//worker.org.gradle.process.internal.worker.GradleWorkerMain.main(GradleWorkerMain.java:74)

````